README.md

```diff
@@ -1,20 +1,56 @@
 <p align="center">
-	<img width=256px src="GitHub/logo.png" />
+    <img width=256px src="GitHub/logo.png" />
 </p>
 
 # Dynamic FPS
 
-Dynamic FPS automatically reduces the speed at which Minecraft renders when it's not focused (to 1 FPS) or hidden (no renders at all). It also fixes a bug in Vanilla Minecraft that makes it take much more performance in the background than it should.
+Dynamic FPS can reduce the frame rate, volume, and pause toasts while Minecraft is not focused or hidden.  
+In addition to this it also fixes a bug in vanilla Minecraft making it take too many resources in the background.
 
-Download in [the releases section](https://github.com/juliand665/Dynamic-FPS/releases), on [Modrinth](https://modrinth.com/mod/dynamic-fps), or on [CurseForge](https://www.curseforge.com/minecraft/mc-mods/dynamic-fps).
+Download on [GitHub Releases](https://github.com/juliand665/Dynamic-FPS/releases), [Modrinth](https://modrinth.com/mod/dynamic-fps), or [CurseForge](https://www.curseforge.com/minecraft/mc-mods/dynamic-fps).
+
+## Features
+
+Dynamic FPS will detect when the window is not focused, but instead either hovered over, unfocused, or invisible.  
+For each mode you can adjust the maximum FPS, master volume, toast visibility, and whether to invoke garbage collection.
+
+In addition to this Dynamic FPS also reduces background CPU usage and stops rendering the world while either the loading overlay is active or a screen is obscuring the world (you're in eg. the keybinds or any detected screen).
+
+Other than that Dynamic FPS is kept very light-weight - no update checker, no telemetry, or anything else unneeded!
+
+## Questions
+
+- Why is Minecraft still running at 15 FPS?
+
+Dynamic FPS will only reduce the client / render loop to a minimum of 15 cycles per second.
+Lower frame rates are achieved by then cancelling the rendering of all superfluous frames, eg. 14 out of 15 frames are cancelled for 1 FPS.
+
+This is done to make resume playing feel pretty much instantaneous - instead of being able to
```

Comment:
i'd suggest an em dash like "instantaneous—instead" rather than a hyphen here. not sure if github renders `--` in markdown as —?

also, "to make resume playing" feels weird; i'd try "This lets you resume playing almost instantly after switching back to the game: Instead of having to wait up to a second for the next frame at 1 FPS, you only wait up to 66ms for the next check to run."

Reply:
Hmm. I'm not sure if I like removing where we get this magic number of 66ms from. What do you think of this instead:
`This lets you resume playing almost instantly after switching back to the game:
Instead of having to wait for up to a second for the next frame you only have to wait one 15th of a second for the next check to run.`

Reply:
I like the idea! Now that I'm looking at it some more, perhaps some reordering would help clarify: `This lets you resume playing almost instantly after switching back to the game: Instead of having to wait for up to a second until the next rendered frame comes along, the game checks back within 1/15th of a second.`